extensions/roc-plugin-style-css/src/roc/index.js

```diff
@@ -25,5 +25,15 @@ export default {
             hasCallback: true,
             returns: isObject(oneOf(isString, isArray(isString))),
         },
+        'add-style-preloaders': {
+            description: `
+            Used to preloaders to style.
```

Comment:
I think we should change the description to better describe what the hook does.

Maybe something like the following?

```
Used to add general loaders early in the chain, before the PostCSS loader.

These loaders will be applied to all styles added from the \`add-style\` hook.
```

We don't need the the description of the return value, that is given based on the validator.